### DISCUSSION TITLE
Snowflake?

### DISCUSSION THREAD

**Discussion by abel-diaz** [sfu-db/connector-x#189] [General]
Hi, Im loving CX in Oracle. Will CX be available for Snowflake? If so around when? Thanks for all the hard work in making this!
Best,
Abel

**Comment by wangxiaoying** [comment]
Hi @abel-diaz , thanks for open this discussion! We do want to support Snowflake in the future. However, it looks like for now there is no Rust driver that we could directly adopt and their wire protocol is not open-sourced. So the only option for us is to port one of their [provided drivers](https://docs.snowflake.com/en/user-guide/conns-drivers.html), which needs more investigation. We may need some time to evaluate the cost for develop and maintenance compare with the need.
